src/features/home/screen/index.tsx

```diff
@@ -1,13 +1,33 @@
 import React from 'react';
 import { SafeAreaView, Text } from 'react-native';
 
-import styles from './styles';
+import { useNavigation } from '@react-navigation/native';
+
+import Button from 'common/Button';
+
+import { MainStackScreens } from 'navigation/stacks/main';
+
+import useStyles from './styles';
 import type { HomeNavigationProps } from './types';
 
-const HomeScreen: React.FunctionComponent<HomeNavigationProps> = () => (
-  <SafeAreaView style={styles.container}>
-    <Text accessibilityRole={'text'}>Home Screen</Text>
-  </SafeAreaView>
-);
+const HomeScreen: React.FunctionComponent<HomeNavigationProps> = () => {
+  const styles = useStyles();
+  const { navigate } = useNavigation();
+  const goToSettings = () => navigate(MainStackScreens.Settings);
+
+  return (
+    <SafeAreaView style={styles.container}>
+      <Text accessibilityRole="text" style={styles.title}>
+        Home Screen
```

Comment:
This string could be moved to the locale :)

Reply:
Done!